hazelcast/src/hazelcast/logger.cpp

```diff
@@ -53,6 +54,22 @@ void logger::log(const char *file_name, int line,
     handler_(instance_name_, cluster_name_, file_name, line, lvl, msg);
 }
 
+namespace {
+
```

Comment:
can we just utilize utilities from boost and not have an extra method? did you check boost?  e.g. https://stackoverflow.com/questions/2492775/get-local-time-with-boost  The `boost::posix_time::second_clock::local_time()` uses `localtime_r`

Reply:
We decided to do this later before 4.0 release. An issue will be opened for this.